### ISSUE TITLE
Strike and Dip Measurement Tool Auto Measure Not Functioning

### ISSUE THREAD

**Issue by amfintel** [StraboSpot/StraboField#441]
I am a bit unfamiliar with GitHub and have only recently downloaded StraboSpot2 so I apologize if this is an incorrect way to report this bug. 

I have run into an issue with the measurement tool for strike and dip measurements. The auto measure tool does not register the iPad's compass or orientation. It shows a blank compass and if you click it to take a measurement, it just puts a ?/? planar feature measurement into the spot. 

I have uninstalled and reinstalled the app. I have tested the compass and measurement abilities of the iPad with other apps (Such as clino) and they function properly. I tested the measurement tool on an older generation iPad and it functions perfectly but I am unsure if the older iPad has the most updated version of StraboSpot2. It appears the measurement tool has slight visual differences from my app on the new iPad to my colleague's on the older iPad. I noticed an update about two weeks ago relating to the compass design and I have only been using it on my iPad for the last week so I have no experience with the app before this most recent update. 

I am using an iPad Pro 11 inch 3rd generation, Software version 15.4.1

I have attached a screenshot of the StraboSpot2 view while trying to use the measurement tool
![StraboSpot2_Measurement_Issue](https://user-images.githubusercontent.com/81640771/171483760-b693b099-079a-4903-b112-dcb10fc8b03d.png)

**Comment by NathanNovak** [comment]
Hi @amfintel What version of the app are you running? You can find that in Home => About

**Comment by amfintel** [comment]
I opened the app, started a new project, placed a spot and hit the take a measurement button, that’s what brought me to the current view above.  Alternatively I’ve tried to add the measurement tool as a shortcut to the main map and use that, it still shows the same as before with no measurement.

**Comment by NathanNovak** [comment]
And this is still happening after you reinstalled the app?

Nathan Novak
520-400-5793

**Comment by amfintel** [comment]
Yes, I have uninstalled and reinstalled it three times. I’ve deleted all the files, restarted the iPad with the app installed, restarted it without the app installed. Still no change, I have also tried proceeding as guest to test it instead of logging in. I have allowed location “only once” and “while app is in use”. It is locating me accurately. I’ve confirmed compass is turned on under navigation in apple settings.

**Comment by NathanNovak** [comment]
Ok we will do some more testing to see if the issue can be replicated on this end.  I am assuming you were not connected to wifi or cellular?

**Comment by amfintel** [comment]
I found the issue! It was something on my end. I tried connected to wifi and not connected. cellular has not been connected or turned on yet. However, I checked through cellular features and found that the 'compass calibration' setting was turned off in Location Services > System Services. It appears it's off by default if a new iPad with cellular capabilities has not had cellular activated yet. It's one of those hidden settings that does not show up in settings searches for 'compass'. It appears to be working fine now, sorry for all the trouble!

**Comment by NathanNovak** [comment]
Ok I'll take a look
